README.md

```diff
@@ -1,8 +1,230 @@
+    <br />
+    <a href="https://github.com/annel18/air-quality-monitoring"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/annel18/air-quality-monitoring">View Demo</a>
+    ·
+    <a href="https://github.com/annel18/air-quality-monitoring/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/annel18/air-quality-monitoring/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Built With
+* [![nodejs][Node.js]][nodejs-url]
+* [![React][React.js]][React-url]
+* [![Vite][Vite.dev]][Vite-url]
+* [![Sass][Sass]][Sass-url]
+* [![Bootstrap][Bootstrap.com]][Bootstrap-url]
+* [![Jest][Jest.js]][Jest-url]
+* [![testing-library][testing-library]][ReactTesting-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+<!-- This is an example of how you may give instructions on setting up your project locally. -->
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+1. Get a free API Key at [https://aqicn.org/data-platform/token/](https://aqicn.org/data-platform/token/)
+2. Clone the repo
+   ```sh
+   git clone https://github.com/annel18/air-quality-monitoring.git
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+4. Enter your API in `.env`
+   ```env
+    API_KEY = 'ENTER YOUR API';
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+
+_For more examples, please refer to the [Documentation](https://example.com)_
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
can you add some testing info? which library, how to run tests, what was the test coverage? do you need a mocking? which mocking library , how will you test third-party rest-apis (just testing you won't hit the real rest endpoint I guess) etc..

Reply:
New testing section added